packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -44,6 +55,13 @@ export default class VotePopup extends Vue {
     this.openProposalDialog = true;
   }
 
+  async getUserNodes(farmIDs: any[]): Promise<any> {
```

Comment:
we already know the type of the farmIDs
```suggestion
  async getUserNodes(farmIDs: number[]): Promise<any> {
```